### ISSUE TITLE
Include/Exclude Apps List

### ISSUE THREAD

**Issue by styromaniac** [yggdrasil-network/crispa-android#29]
Some apps don't work while Yggdrasil is enabled. Including/Excluding apps from Yggdrasil is a good solution, right?

**Comment by styromaniac** [comment]
I see that it's unusual for apps to not have internet connectivity, so I decided to close this.

**Comment by ChronosXYZ** [comment]
> Some apps don't work while Yggdrasil is enabled. Including/Excluding apps from Yggdrasil is a good solution, right?

@styromaniac What do you mean by including/excluding apps? It doesn't intersects with the regular Internet IP range, and you can browse the Internet while running Yggdrasil daemon. Can you give some example when it interfere with specific apps?

**Comment by styromaniac** [comment]
Yes. MeWe is affected. A new post's image(s) won't load with Yggdrasil enabled.

Freetone too. Can't call or receive calls with Yggdrasil.

Videoder can't load videos. https://videoder.net

**Comment by styromaniac** [comment]
Every app seems to work if IPv6 is supported by the ISP. An IPv4-only ISP may reveal the issue to you.

**Comment by styromaniac** [comment]
Apparently I didn't have an IPv6 address by the ISP for a moment and I couldn't speed test with Yggdrasil enabled.

**Comment by ChronosXYZ** [comment]
Yggdrasil doesn't require IPv6 support by ISP.

**Comment by styromaniac** [comment]
> Yggdrasil doesn't require IPv6 support by ISP.

I know that should be, but I'm seeing these apps and then some only performing on Verizon and not on MIcom while Yggdrasil is enabled.

I hadn't thought to test with an AT&T hotspot yet, but they have IPv6 support, so I'm expecting all applications to work.

**Comment by ChronosXYZ** [comment]
> > Yggdrasil doesn't require IPv6 support by ISP.
> 
> I know that should be, but I'm seeing these apps and then some only performing on Verizon and not on MIcom while Yggdrasil is enabled.
> 
> I hadn't thought to test with an AT&T hotspot yet, but they have IPv6 support, so I'm expecting all applications to work.

I have ISP which doesn't support IPv6, and all apps work correctly. I need logs of Yggdrasil app to identify the problem.

**Comment by styromaniac** [comment]
> > > Yggdrasil doesn't require IPv6 support by ISP.
> > 
> > 
> > I know that should be, but I'm seeing these apps and then some only performing on Verizon and not on MIcom while Yggdrasil is enabled.
> > I hadn't thought to test with an AT&T hotspot yet, but they have IPv6 support, so I'm expecting all applications to work.
> 
> I have ISP which doesn't support IPv6, and all apps work correctly.

The apps I had issues with?

**Comment by styromaniac** [comment]
I was wrong and you're right. It has to be something else. Charter is Ipv4-only and it's working with Yggdrasil.

MIcom is the one with the problem.

**Comment by ChronosXYZ** [comment]
We can implement that feature by `VPNService.Builder.addDisallowedApplication` method.

**Comment by styromaniac** [comment]
#34
